S/Silo/build_tarballs.jl

```diff
@@ -83,4 +83,4 @@ ENV["MPITRAMPOLINE_DELAY_INIT"] = "1"
 
 # We need to use at least GCC 8 to ensure that we get at least libgfortran5, which we need for HDF5.
 build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies;
-               julia_compat="1.6", preferred_gcc_version=v"8")
+               augment_platform_block, julia_compat="1.6", preferred_gcc_version=v"8")
```

Comment:
Thinking about it, for packages which use the libgfortran ABI (and downstream packages, too) actually require Julia v1.7, or maybe let's go to v1.10 to be safe
```suggestion
               augment_platform_block, julia_compat="1.10", preferred_gcc_version=v"8")
```